src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-all-attributes.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: Remove all attributes
+type: apiDoc
+shortDescription: 'Removes all attributes.'
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to remove all attributes.
+redirects:
+- /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/removeallattributes-ios-sdk-api
+- /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/remove-all-attributes
+- /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/removeallattributes-android-agent-api
+- /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/removeallattributes-android-sdk-api
+- /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/remove-all-attributes
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="cordova">
+            Cordova
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React Native
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.removeAllAttributes ()
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+NewRelic.removeAllAttributes()
+```
+
+## Description [#description]
+
+This method removes all attributes from a session. If you want to remove a single session attribute, see [Remove an attribute](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-all-attributes).
+
+For context on how to use this API, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api#attributes-events-insights).
+
+## Return values [#return-values]
+
+Returns `true` if recorded successfully, and `false` if it doesn't.
+
+## Example [#example]
+
+### Java [#java]
+
+```java
+boolean attributesRemoved = NewRelic.removeAllAttributes();
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
val attributesRemoved = NewRelic.removeAllAttributes()

Reply:
```suggestion
```kotlin
    val attributesRemoved = NewRelic.removeAllAttributes()
```
```